### PR TITLE
Editorial: remove conflicting exports

### LINKED ISSUE #1509: [meta] Please mark terms with standard definitions elsewhere as noexport
In <https://w3c.github.io/aria/#terms>, a large number of terms are defined that are already well-defined by the web platform elsewhere. Having local versions of these definitions, with descriptions that are more useful/specialized for readers of this specific spec, is fine, but by marking them all as `export` they're conflicting with definitions elsewhere in the web platform that are otherwise defined in only a single location, like "Event".

(This was not previously an issue for Bikeshedded specs, as the data source I was using didn't include this document, but I'm in the process of switching to a new data source that does.)

It would be great to have this entire list switched to noexport, with only terms for which this spec is actually a primary definition source marked as export. For example, "Accessibility API" might be appropriate to still export, but "attribute", "class", "DOMString", etc are not.

(Ideally these terms are also marked more explicitly as "local-only", by setting an empty `lt` attribute and putting the linking text into `local-lt`, like `<dfn lt local-lt="attribute">`. But at least marking them as unexported will avoid annoying conflicts across the entire web platform once I kick the data source over.)

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #1509

### Summary
This PR does the following:
* Moves definition for Accessible Description, Accessible Name and Tooltip Attribute definitions  to Accname
* Makes definitons for the following not exported (cannot be used in other specs)
  * Deprecated
  * Desktop Focus
  * Expose
  * Event
  * Graphical Document
  * Keyboard Accessible
  * Landmark
  * Nemeth Braille
  * Object
  * Ontology
  * Operable
  * Owned Element
  * Owning Element
  * Property
  * Target Element
* removes definitions for the following
  * Attribute (refer instead to [=attribute=] concept)
  * Class (remove references to this)
  * DOMString (allow to reference WebIDL)
  * Element (refer to [=element=] concept
  * Informative (remove references to this)
  * Primary Content Element (was not used)
  * Node (remove reference or reference [=nodes=] concept if a DOM node)
  * Normative (remove references to this)
  * Root WAI-ARIA Node (was not used)
  * Sub-document (was not used)
  * Taxonomy (was not used)
  * Text Node (was only used in definition of Node)
  * Underestandable (was not used)
  * User Agent (refer to [=user agent=] concept
  * Valid IDREF (does not need to be defined - remove references to it)
* Requries hidden references to be changed to [=element/hidden=]

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria/pull/1510.html" title="Last updated on Jul 22, 2021, 2:22 PM UTC (6192ccd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/1510/b41a010...6192ccd.html" title="Last updated on Jul 22, 2021, 2:22 PM UTC (6192ccd)">Diff</a>

***

### :boom: Error: 504 Gateway Timeout :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on May 27, 2022, 11:49 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Faria%2F28dad1d93b4649b8f464ccfde65731b0c109700e%2Findex.html%3FisPreview%3Dtrue)

```
<html><body><h1>504 Gateway Time-out</h1>
The server didn't respond in time.
</body></html>

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/aria%231510.)._
</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -249,25 +249,25 @@ <h1>Important Terms</h1>
 	<div>
 		<p>While some terms are defined in place, the following definitions are used throughout this document. </p>
 		<dl class="termlist">
-			<dt><dfn data-export=""  data-lt="accessibility api|accessibility apis">Accessibility <abbr title="Application Programming Interface">API</abbr></dfn></dt>
+			<dt><dfn data-lt="accessibility api|accessibility apis">Accessibility <abbr title="Application Programming Interface">API</abbr></dfn></dt>
```
Comment (jnurthen) [inline index.html:252]: This would break all the other specs in the ARIA suite. Core-AAM for example references this definition 12 times. Other specs (Accname, dpub-aria, dpub-aam etc.) all reference this too. How would they then reference this term?

File: index.html [hunk 2]
```diff
@@ -249,25 +249,25 @@ <h1>Important Terms</h1>
...
 				  <a href="https://developer.apple.com/documentation/appkit/nsaccessibility">Mac <abbr title="OS Ten">OS X</abbr> Accessibility Protocol</a> [[AXAPI]], the <cite><a href="https://developer.gnome.org/atk/unstable/">Linux/Unix Accessibility Toolkit</a></cite> [[ATK]] and <cite><a href="https://developer.gnome.org/libatspi/stable/">Assistive Technology Service Provider Interface</a></cite> [[AT-SPI]], and <a href="https://wiki.linuxfoundation.org/accessibility/iaccessible2/start">IAccessible2</a> [[IAccessible2]].</p>
 			</dd>
-			<dt><dfn data-export="" >Accessible Description</dfn></dt>
+			<dt><dfn >Accessible Description</dfn></dt>
```
Comment (jnurthen) [inline index.html:257]: Why can't we export Accessible Description? This seems to be something we can certainly define.

File: index.html [hunk 3]
```diff
@@ -249,25 +249,25 @@ <h1>Important Terms</h1>
...
 			<dd>
 			  <p>An accessible description provides additional information, related to an interface element, that complements the <a>accessible name</a>. The accessible description might or might not be visually perceivable. </p>
 			</dd>
-			<dt><dfn data-export="" data-lt="accessible name|accessible names">Accessible Name</dfn></dt>
+			<dt><dfn data-lt="accessible name|accessible names">Accessible Name</dfn></dt>
```
Comment (jnurthen) [inline index.html:261]: same for accessible name

File: index.html [hunk 4]
```diff
@@ -249,25 +249,25 @@ <h1>Important Terms</h1>
...
 			  <p>A simple use for the accessible name property may be illustrated by an &quot;OK&quot; button. The text &quot;OK&quot; is the accessible name. When the button receives focus, assistive technologies may concatenate the platform's role description with the accessible name. For example, a screen reader may speak &quot;push-button OK&quot; or &quot;OK button&quot;. The order of concatenation and specifics of the role description (e.g., &quot;button&quot;, &quot;push-button&quot;, &quot;clickable button&quot;) are determined by platform <a class="termref" data-lt="accessibility api">accessibility API</a>s or <a>assistive technologies</a>.</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="accessible object|accessible objects">Accessible object</dfn></dt>
+			<dt><dfn data-lt="accessible object|accessible objects">Accessible object</dfn></dt>
```
Comment (jnurthen) [inline index.html:266]: accessible object too

**Review by jnurthen — CHANGES_REQUESTED** [review]
I'm really not sure how this can work. We have multiple specs (actually from multiple groups). We removed the central terms definition logic at the request of respec - but now this replacement technique doesn't seem to work. 
Simply removing the exports doesn't work for the ARIA group as it will break all the dependent specs

File: index.html [hunk 5]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
 			<dd>
 			  <p>Main indicator of type. <!-- (removing, vague) The object's role is the class of <a class="termref" data-lt="object">objects</a> of which it is a member. --> This <a class="termref" data-lt="semantics">semantic</a> association allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.</p>
 			</dd>
-			<dt><dfn data-export="">Root <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> node</dfn></dt><!--Not used-->
+			<dt><dfn>Root <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> node</dfn></dt><!--Not used-->
 			<dd>
 			  <p> The primary element containing non-metadata content. In many languages, this is  the document element but in <abbr title="Hypertext Markup Language">HTML</abbr>, it is the <code>&lt;body&gt;</code>.</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="semantics|semantic|semantically">Semantics</dfn></dt>
+			<dt><dfn data-lt="semantics|semantic|semantically">Semantics</dfn></dt>
```
Comment (jnurthen) [inline index.html:396]: Needs export

File: index.html [hunk 6]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
 			<dd>
-			  <p>The meaning of something as understood by a human, defined in a way that computers can process a representation of an <a>object</a>, such as <a class="termref" data-lt="element">elements</a> and <a class="termref" data-lt="attribute">attributes</a>, and reliably represent the object in a way that various humans will achieve a mutually consistent understanding of the object.</p>
+			  <p>The meaning of something as understood by a human, defined in a way that computers can process a representation of an <a>object</a>, such as <a class="termref" data-lt="element">elements</a> and [=attribute|attributes=], and reliably represent the object in a way that various humans will achieve a mutually consistent understanding of the object.</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="state|states">State</dfn></dt>
+			<dt><dfn data-lt="state|states">State</dfn></dt>
```
Comment (jnurthen) [inline index.html:400]: Needs export

File: index.html [hunk 7]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
 			<dd>
 			  <p>A state is a dynamic <a class="termref" href="#dfn-property">property</a> expressing characteristics of an <a>object</a> that may change in response to user action or automated processes. States do not affect the essential nature of the object, but represent data associated with the object or user interaction possibilities. See <a class="specref" href="#statevsprop">clarification of states versus properties</a>.</p>
 			</dd>
-			<dt><dfn data-export="">Sub-document</dfn></dt><!--Not used-->
+			<dt><dfn>Sub-document</dfn></dt><!--Not used-->
```
Comment (jnurthen) [inline index.html:404]: As the comment says this is not used!

File: index.html [hunk 8]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
 			  <p>An element specified in a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> relation. For example, in <code> &lt;div aria-controls=”elem1”&gt;</code>, where <code>“elem1”</code> is the ID for the target element.</p>
 			</dd>
-			<dt><dfn data-export="">Taxonomy</dfn></dt><!--Not used-->
+			<dt><dfn>Taxonomy</dfn></dt><!--Not used-->
 			<dd>
-			  <p>A hierarchical definition of how the characteristics of various <a class="termref" data-lt="class">classes</a> relate to each other, in which classes inherit the properties of superclasses in the hierarchy. A taxonomy can comprise part of the formal definition of an <a>ontology</a>.</p>
+			  <p>A hierarchical definition of how the characteristics of various classes relate to each other, in which classes inherit the properties of superclasses in the hierarchy. A taxonomy can comprise part of the formal definition of an <a>ontology</a>.</p>
 			</dd>
-			<dt><dfn data-export="">Text node</dfn></dt>
+			<dt><dfn>Text node</dfn></dt>
```
Comment (jnurthen) [inline index.html:416]: There has to be a DOM definition for this

File: index.html [hunk 9]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
 			<dd>
 			  <p>Type of <abbr title="Document Object Model">DOM</abbr> <a class="termref" data-lt="node">node</a> that represents the textual content of an <a class="termref">attribute</a> or an <a class="termref">element</a>. A Text node has no child nodes.  </p>
 			</dd>
-			<dt><dfn data-export="">Tooltip attribute</dfn></dt><!--Accname-->
+			<dt><dfn>Tooltip attribute</dfn></dt><!--Accname-->
```
Comment (jnurthen) [inline index.html:420]: need to move this to accname - then no export needed

File: index.html [hunk 10]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
 			<dd>
 			  <p>Any host language attribute that would result in a user agent generating a tooltip such as in response to a mouse hover in desktop user agents.</p>
 			</dd>
-			<dt><dfn data-export="">Understandable</dfn></dt><!--Not used-->
+			<dt><dfn>Understandable</dfn></dt><!--Not used-->
```
Comment (jnurthen) [inline index.html:424]: don't think this is needed.

File: index.html [hunk 11]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
 			</dd>
-			<dt><dfn data-export="" data-lt="unicode braille">Unicode Braille Patterns</dfn></dt>
+			<dt><dfn data-lt="unicode braille">Unicode Braille Patterns</dfn></dt>
 			<dd>
 				<p>In Unicode, braille is represented in a block called Braille Patterns (U+2800..U+28FF). The block contains all 256 possible patterns of an 8-dot braille cell; this includes the complete 6-dot cell range which is represented by U+2800..U+283F. In all braille systems, the braille pattern dots-0 (U+2800) is used to represent a space or the lack of content; it is also called a blank Braille pattern. See <cite><a href="https://en.wikipedia.org/wiki/Braille_Patterns">Braille Patterns on Wikipedia</a>.</cite></p>
 			</dd>
-			<dt><dfn data-export="" data-lt="user agent|user agents">User Agent</dfn></dt>
+			<dt><dfn data-lt="user agent|user agents">User Agent</dfn></dt>
```
Comment (jnurthen) [inline index.html:432]: Needs export

File: index.html [hunk 12]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
 			<dd>
 			  <p>Any software that retrieves, renders and facilitates end user interaction with Web content. This definition may differ from that used in other documents.</p>
 			</dd>
-			<dt><dfn data-export="">Valid IDREF</dfn></dt>
+			<dt><dfn>Valid IDREF</dfn></dt>
```
Comment (jnurthen) [inline index.html:436]: not sure we need to define this at all. IMO we should remove the one reference to it and delete the definition

File: index.html [hunk 13]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
 			<dd>
 			  <p>A reference to a <a>target element</a> in the same document that has a matching ID</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="widget|widgets">Widget</dfn></dt>
+			<dt><dfn data-lt="widget|widgets">Widget</dfn></dt>
```
Comment (jnurthen) [inline index.html:440]: needs export

**Review by jnurthen — COMMENTED** [review]
We're going to need to create similar PRs for accname, core-aam, dpub-aria, dpub-aam, html-aam too (although there should be fewer instances in those specs)

File: index.html [hunk 14]
```diff
@@ -244,30 +244,30 @@ <h2>Assistive Technologies</h2>
...
 	</section>
 </section>
-<section class="informative" id="terms">
+<section class="informative" id="terms" data-cite="dom">
 	<h1>Important Terms</h1>
 	<div>
 		<p>While some terms are defined in place, the following definitions are used throughout this document. </p>
 		<dl class="termlist">
-			<dt><dfn data-export=""  data-lt="accessibility api|accessibility apis">Accessibility <abbr title="Application Programming Interface">API</abbr></dfn></dt>
+			<dt><dfn>Accessibility <abbr title="Application Programming Interface">API</abbr></dfn></dt>
```
[the remark this answers is not in the document]
Comment (marcoscaceres) [reply index.html:252]: oops! good catch.

Comment (marcoscaceres) [inline index.html:252]:
```suggestion
			<dt><dfn data-export="">Accessibility <abbr title="Application Programming Interface">API</abbr></dfn></dt>
```

File: index.html [hunk 15]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
 			</dd>
-			<dt><dfn data-export="" data-lt="live region|live regions">Live Region</dfn></dt>
+			<dt><dfn data-lt="live region|live regions">Live Region</dfn></dt>
 			<dd>
 			  <p>Live regions are perceivable regions of a web page that are typically updated as a result of an external event when user focus may be elsewhere. These regions are not always updated as a result of a user interaction. Examples of live regions include a chat log, stock ticker, or a sport scoring section that updates periodically to reflect game statistics. Since these asynchronous areas are expected to update outside the user's area of focus, assistive technologies such as screen readers have either been unaware of their existence or unable to process them for the user. WAI-ARIA has provided a collection of properties that allow the author to identify these live regions and process them: aria-live, aria-relevant, aria-atomic, and aria-busy.</p>
 			</dd>
-			<dt><dfn data-export="">Primary Content Element</dfn></dt><!--Not used-->
+			<dt><dfn>Primary Content Element</dfn></dt><!--Not used-->
```
[the remark this answers is not in the document]
Comment (marcoscaceres) [reply index.html:340]: Will remove. Can probably hook into firstElementChild if needed elsewhere.

File: index.html (hunk 11 above)
Comment (marcoscaceres) [reply index.html:432]: This is defined by Infra, so should be probably be dropped. 
https://infra.spec.whatwg.org/#user-agent

Comment (marcoscaceres) [reply index.html:432]: yeah, the definition at Infra is almost identical to this one, so we should be good to drop this one.

File: index.html [hunk 16]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
 			<dd>
 			  <p><a>Accessibility API</a> <a>state</a> that is controlled by the user agent, such as focus and selection. These are contrasted with &quot;unmanaged states&quot; that are typically controlled by the author. Nevertheless, authors can override some managed states, such as aria-posinset and aria-setsize. Many managed states have corresponding CSS pseudo-classes, such as :focus, and pseudo-elements, such as ::selection, that are also updated by the user agent.</p>
 			</dd>
-			<dt><dfn data-export="">Nemeth Braille</dfn></dt>
+			<dt><dfn>Nemeth Braille</dfn></dt>
 			<dd>
 				<p>The Nemeth Braille Code for Mathematics is a braille code for encoding mathematical and scientific notation. See <cite><a href="https://en.wikipedia.org/wiki/Nemeth_Braille">Nemeth Braille on Wikipedia</a>.</cite></p>
 			</dd>
-			<dt><dfn data-export="">Node</dfn></dt>
+			<dt><dfn>Node</dfn></dt>
```
[the remark this answers is not in the document]
Comment (marcoscaceres) [reply index.html:348]: Ok, let's figure this one out tomorrow... I wonder if we can define "accessible node"? Or maybe some other spec does it?

File: index.html (hunk 9 above)
Comment (marcoscaceres) [reply index.html:416]: Ok, I wasn't sure where to put it in Accname. Could you send a PR for it?

File: index.html [hunk 17]
```diff
@@ -152,7 +152,7 @@
 <section id="sotd">
 	<p>The Accessible Rich Internet Applications Working Group seeks feedback on any aspect of the specification. When submitting feedback, please consider issues in the context of the companion documents. To comment, <a href="https://github.com/w3c/aria/issues/new">file an issue in the <abbr title="World Wide Web Consortium">W3C</abbr> ARIA GitHub repository</a>. If this is not feasible, send email to <a href="mailto:public-aria@w3.org?subject=Comment%20on%20WAI-ARIA%201.2">public-aria@w3.org</a> (<a href="http://lists.w3.org/Archives/Public/public-aria/">comment archive</a>). In-progress updates to the document may be viewed in the <a href="http://w3c.github.io/aria/">publicly visible editors' draft</a>.</p>
 </section>
-<section class="informative" id="introduction">
+<section class="informative" id="introduction" data-cite="dom">
```
Comment (marcoscaceres) [inline index.html:155]: Can we start making these concepts normative? As they are used by the rest of the spec in normative parts, that would make sense. 

 
```suggestion
<section  id="introduction" data-cite="dom">
```

File: index.html [hunk 18]
```diff
@@ -244,30 +244,30 @@ <h2>Assistive Technologies</h2>
...
 				  <a href="https://developer.apple.com/documentation/appkit/nsaccessibility">Mac <abbr title="OS Ten">OS X</abbr> Accessibility Protocol</a> [[AXAPI]], the <cite><a href="https://developer.gnome.org/atk/unstable/">Linux/Unix Accessibility Toolkit</a></cite> [[ATK]] and <cite><a href="https://developer.gnome.org/libatspi/stable/">Assistive Technology Service Provider Interface</a></cite> [[AT-SPI]], and <a href="https://wiki.linuxfoundation.org/accessibility/iaccessible2/start">IAccessible2</a> [[IAccessible2]].</p>
 			</dd>
-			<dt><dfn data-export="" >Accessible Description</dfn></dt>
+			<dt><dfn data-export="">Accessible Description</dfn></dt>
 			<dd>
 			  <p>An accessible description provides additional information, related to an interface element, that complements the <a>accessible name</a>. The accessible description might or might not be visually perceivable. </p>
 			</dd>
-			<dt><dfn data-export="" data-lt="accessible name|accessible names">Accessible Name</dfn></dt>
+			<dt><dfn data-export="">Accessible Name</dfn></dt>
```
Comment (marcoscaceres) [inline index.html:261]: Move to accname? Seems like the definitive place for it.

File: index.html [hunk 19]
```diff
@@ -244,30 +244,30 @@ <h2>Assistive Technologies</h2>
...
 			</dd>
-			<dt><dfn data-export="" data-lt="accessible object|accessible objects">Accessible object</dfn></dt>
-			<dd>
-			  <p>A <a>node</a> in the <a class="termref">accessibility tree</a> of a platform <a>accessibility <abbr title="application programming interface">API</abbr></a>. Accessible objects expose various <a class="termref" data-lt="state">states</a>, <a class="termref" data-lt="property">properties</a>, and <a class="termref" data-lt="event">events</a> for use by <a>assistive technologies</a>.  In the context of markup languages (e.g., HTML and SVG) in general, and of WAI-ARIA in particular, markup <a class="termref" data-lt="element">elements</a> and their <a class="termref" data-lt="attribute">attributes</a> are represented as accessible objects.  </p>
+			<dt><dfn data-export="">Accessible object</dfn></dt>
```
Comment (marcoscaceres) [inline index.html:266]: Would be great to rename this one: 
```suggestion
			<dt><dfn data-export="">Accessible node</dfn></dt>
```

As it's a type of node.

File: index.html (hunk 17 above)
Comment (marcoscaceres) [reply index.html:155]: Alternatively, we could split this into normative, informative terms if need be... but probably just fine to treat terms as normative.

**Comment by marcoscaceres** [comment]
Looks like "ontology" could go too, possibly some others...

File: index.html (hunk 5 above)
Comment (marcoscaceres) [reply index.html:388]: Exporting. Maybe reconsider this one... it's seems to be mostly using the generally accepted definition of "semantics" per the English language.

File: index.html (hunk 6 above)
Comment (marcoscaceres) [reply index.html:392]: Exporting. I think HTML might have something similar. 
https://html.spec.whatwg.org/#states-of-the-type-attribute

We should try to align those.

File: index.html [hunk 20]
```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
...
+			  <p>[=attribute|attributes=] that are essential to the nature of a given <a>object</a>, or that represent a data value associated with the object. A change of a property may significantly impact the meaning or presentation of an object. Certain properties (for example, <pref>aria-multiline</pref>) are less likely to change than <a class="termref" href="#dfn-state">states</a>, but note that the frequency of change difference is not a rule. A few properties, such as <pref>aria-activedescendant</pref>, <pref>aria-valuenow</pref>, and <pref>aria-valuetext</pref> are expected to change often. See <a class="specref" href="#statevsprop">clarification of states versus properties</a>.</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="relationship|relationships">Relationship</dfn></dt>
+			<dt><dfn data-lt="relationship|relationships">Relationship</dfn></dt>
```
[the remark this answers is not in the document]
Comment (marcoscaceres) [reply index.html:376]: This one also seems to be just redefining a common English word?

**Comment by marcoscaceres** [comment]
Ok, I think that addresses all the main ones. This is looking pretty good! 

I would encourage you not to export common English words tho (e.g., semantics, relationship). 

Also, "Nemeth Braille" can probably be removed and the one reference link to Wikipedia.

As a followup, my recommendation would also be to remove Owned Element, Owning Element and just use DOM terminology.

**Comment by marcoscaceres** [comment]
Oh yes, and let me know what y'all decide about "event". That's the only blocker, I think.

File: index.html (hunk 17 above)
Comment (jnurthen) [reply index.html:155]: This is a bigger issue. If we are to consider we need a new issue to be filed and we will deal with it there. This PR should be as limited as we can to unblock or we will never merge it

File: index.html (hunk 19 above)
Comment (jnurthen) [reply index.html:266]: This is deliberately object as this is the term used in many accessibility APIs.

File: index.html (hunk 17 above)
Comment (marcoscaceres) [reply index.html:155]: Filed https://github.com/w3c/aria/issues/1513

File: index.html (hunk 19 above)
Comment (marcoscaceres) [reply index.html:257]: Ok, no problem.

File: index.html [hunk 21]
```diff
@@ -13448,11 +13376,11 @@ <h3>Roles</h3>
...
 	</section>
-	<section id="document-handling_author-errors_states-properties">
+	<section id="document-handling_author-errors_states-properties" data-cite="dom">
 		<h3>States and Properties</h3>
-			<p>In general, <a>user agents</a> do not do much validation of <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> <a class="termref">properties</a>. User agents MAY do some minor validation on request, such as making sure <a data-lt="valid idref">valid IDs</a> are specified for <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> relations, and enforcing things like <pref>aria-posinset</pref> being within 1 and <pref>aria-setsize</pref>, inclusive. User agents are not  responsible for logical validation, such as the following:</p>
+			<p>In general, [=user agent|user agents=] do not do much validation of <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> <a class="termref">properties</a>. User agents MAY do some minor validation on request and enforce things like <pref>aria-posinset</pref> being within 1 and <pref>aria-setsize</pref>, inclusive. User agents are not  responsible for logical validation, such as the following:</p>
```
Comment (jnurthen) [inline index.html:13381]: Why the change here? We should leave this example

**Comment by jnurthen** [comment]
### Summary
This PR does the following:
* Moves definition for Accessible Description, Accessible Name and Tooltip Attribute definitions  to Accname
* Makes definitons for the following not exported (cannot be used in other specs)
  * Deprecated
  * Desktop Focus
  * Expose
  * Event
  * Graphical Document
  * Keyboard Accessible
  * Landmark
  * Nemeth Braille
  * Object
  * Ontology
  * Operable
  * Owned Element
  * Owning Element
  * Property
  * Target Element
* removes definitions for the following
  * Attribute (refer instead to [=attribute=] concept)
  * Class (remove references to this)
  * DOMString (allow to reference WebIDL)
  * Element (refer to [=element=] concept
  * Informative (remove references to this)
  * Primary Content Element (was not used)
  * Node (remove reference or reference [=nodes=] concept if a DOM node)
  * Normative (remove references to this)
  * Root WAI-ARIA Node (was not used)
  * Sub-document (was not used)
  * Taxonomy (was not used)
  * Text Node (was only used in definition of Node)
  * Underestandable (was not used)
  * User Agent (refer to [=user agent=] concept
  * Valid IDREF (does not need to be defined - remove references to it)
* Requries hidden references to be changed to [=element/hidden=]

**Comment by marcoscaceres** [comment]
Thanks @jnurthen for the summary! I copy/pasted it to the PR description at the top. 

I wonder if we can proceed with this and then continue with the following as a followup? 

Some observations from the summary.

As I mentioned somewhere above, my suggestion would be to not define common English things (unless there is a specific behavior tied to user agents): 

 * Deprecated
 * Expose
 * Nemeth Braille (already points to wikipedia)
 * Operable (already points to WCAG, IIRC - so that spec should own it)
 
Concepts that might be work exposing:
 
 * Desktop Focus (but we might want to double check with input events?)
 * Graphical Document (but maybe we should coordinate with CSS WG)  
 * Keyboard Accessible
 * Landmark

These are overly generic (can fall into the common English usage category):

 * Property
 * Object
 * Ontology

We should defer DOM for (we coordinate with @annevk, if need be):

 * Event
 * Owned Element
 * Owning Element
 * Target Element

File: index.html (hunk 21 above)
Comment (marcoscaceres) [reply index.html:13381]: Oh, I changed this one because we dropped "valid idref" from the Terms. I was hoping that by removing the ",such as...," sub-sentence, the paragraph would still make sense.

**Comment by marcoscaceres** [comment]
ping 🛎

**Comment by marcoscaceres** [comment]
🛎 friendly ping 🙏

**Comment by jnurthen** [comment]
ok - I'm going to merge this (at last) - I think it is an improvement and we can address follow ups later.

**Comment by marcoscaceres** [comment]
Thanks @jnurthen 🙏
